changelog_unreleased/javascript/pr-6984.md

```diff
@@ -0,0 +1,28 @@
+#### Fix formatting of labeled statements with comments ([#6994](https://github.com/prettier/prettier/pull/6984) by [@clement26695](https://github.com/clement26695))
+
+Comments in labeled statements should always be printed as leading comments of the labeled statement
```

Comment:
Can you change this to mention that the issue was that the comment printing was stable/idempotent? And update the example to make that easier to understand – see #6983 where I updated your post.

Reply:
I updated it. Let me know if it's better or if it should be updated once again

Reply:
Thanks, that’s great!